packages/salesforcedx-apex-debugger/package.json

```diff
@@ -33,16 +31,14 @@
     "vscode-debugadapter-testsupport": "1.22.0"
   },
   "scripts": {
+    "vscode:package": "npm prune --production",
```

Comment:
This is not needed since I run the npm prune --production already. Let me clean this up.